#!/usr/bin/env python
PACKAGE = 'local_interface'

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t, str_t

gen = ParameterGenerator()

gen.add("ship_id", int_t, 0, "The id of ship or vehicle", 1, 3);
gen.add("fb_message", str_t, 0, "The fb message as input of SubmitFBMessageApi", "remote control on")
gen.add("is_get_control", bool_t, 0, "Bool type to decide whether get control command from server", True)
gen.add("base_url", str_t, 0, "The base url used in http api request", "http://111.231.84.43:8002/monitor/upper/")
gen.add("speed", double_t, 0, "The pre-setting speed for vehicle", 0.5, 0, 5)
gen.add("heading", double_t, 0, "The heading for vehicle to keep wtih", 0, -3.1415, 3.1415)

exit(gen.generate(PACKAGE, "local_interface", "NetManager"))
